lib/assembly/plugins/quast.py

```diff
@@ -41,9 +41,10 @@ def run(self):
 
         #### Add Contig files ####
         cmd_args += contigfiles
-        cmd_args += ['-l', '"{}"'.format(', '.join([cset.name for cset in contigsets]))]
-
-
+        try:
+            cmd_args += ['-l', '"{}"'.format(', '.join([cset.name for cset in contigsets]))]
+        except (KeyError, TypeError): # No name
+            pass
```

Comment:
With the new contig file type, uploaded contigs don't get named, so quast was breaking trying to get label names from ContigSets.